BitwardenShared/UI/Auth/ProfileSwitcher/ProfileSwitcherView.swift

```diff
@@ -9,13 +9,7 @@ struct ProfileSwitcherView: View {
     @ObservedObject var store: Store<ProfileSwitcherState, ProfileSwitcherAction, ProfileSwitcherEffect>
 
     var body: some View {
-        OffsetObservingScrollView(
-            axes: store.state.isVisible ? .vertical : [],
-            offset: .init(
-                get: { store.state.scrollOffset },
-                set: { store.send(.scrollOffsetChanged($0)) }
-            )
-        ) {
+        ScrollView {
```

Comment:
❓ I thought @phil-livefront noticed that removing the axes ternary ended up with the same result? Curious if you tried that.

```
        OffsetObservingScrollView(
            axes: .vertical,
            offset: .init(
                get: { store.state.scrollOffset },
                set: { store.send(.scrollOffsetChanged($0)) }
            )
        ) {
```

Reply:
Yep, that also works. But if we're not changing the `axes` and not going to use the `scrollOffset` then I'd just use a normal `ScrollView` for this as the `OffsetObservingScrollView` won't add any value to this.

Reply:
I guess I'm confused why we don't need the `scrollOffset` anymore?

Reply:
Maybe there's an issue with the scroll offset currently. But I think the intention of it was to hide the space above the scroll view if you scroll it beyond the top of the content.

| Current | Expected |
| -- | -- |
| <video src="https://github.com/user-attachments/assets/087b67fd-7134-4c2f-9b3f-4af8f72b2694" /> | <video src="https://github.com/user-attachments/assets/f041870a-9234-4826-9dcc-aadde6b226a6" /> |

I can fix this separately, but if it's not causing any issue maybe we should keep it in.